examples/convert_i3_to_sqlite.py

```diff
@@ -60,8 +60,10 @@ def main_icecube_upgrade():
         ],
         outdir,
         gcd_rescue,
-        db_name=db_name,
         workers=workers,
+        nb_files_to_batch=1000,
+        # sequential_batch_pattern="temp_{:03d}",
+        input_file_batch_pattern="[A-Z]{1}_[0-9]{5}*.i3.zst",
```

Comment:
Can we add a comment to explain this pattern? Not clear to me

Reply:
Added to the `DataConverter` docstring.